test: add coverage

brettz9 opened the pull request:
If you don't want the package-lock, I can rebase with a `.gitignore` (or if you prefer `.ncurc`), but I find a lock file helpful, including for the ability to perform `npm audit`.

jhermsmeier commented:
Not sure where to start really – I'm delighted at the barrage of pull requests I just woke up to :)
Indeed I would prefer the `package-lock.json` to be removed – as an `npm audit` can still be done via `npm install --package-lock-only && npm audit` when it's needed (i.e. on CI).
What is an `.ncurc` file – which tool consumes that?

brettz9 commented:
:) Glad to hear it.

Sorry, I meant `.npmrc` file (there is such a thing as a `.ncurc` file too, but it is used instead with the very helpful project, `npm-check-updates` to allow the tool to avoid checking or updating certain dependencies within a package; I used `npm-check-updates` for my npm PR to find out which deps. were outdated)

I've removed `package-lock.json` and added it to `.gitignore`, but let me know if you want it through `.npmrc` instead.

jhermsmeier commented:
Having it in the `.gitignore` is perfect, thanks!
